guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -3522,9 +3520,7 @@ WEBVTT
 
 02:02:46.287 --> 02:03:11.687
 [Tom]: intra-body communication between the nanopods and 
-
 [J Jay Couey]: oh wow yeah okay that's a good one um yeah...
-
 [Tom]: Well wait let me just oh yeah go ahead loop back and say that professor uh Anna Ulreich said no no this is just uh this is just crystallization and it's well documented and then after this everyone yeah why don't you comment i have maybe two more and that's it. 
```

Comment:
After removing the whitespaces, it seems like any number of lines will get rendered as 1 caption.
![image](https://github.com/user-attachments/assets/f4a6a9d5-dfe8-486c-af4e-2f41f645fe10)